packages/sdk/src/modules/did/index.ts

```diff
@@ -152,24 +162,48 @@ export class Did extends Base {
     return `did:peaq:${address}`;
   }
 
-  private _createVerificationMethod(address: Address) {
-    const id = uuidv4();
+  private _createVerificationMethod(verification: Verification, address: Address, keyNum: number) {
+    const id = `did:peaq:${address}#keys-${keyNum}`;
```

Comment:
This id prefix should not be hardcoded into the source.
We should the developer option to use their custom prefix.

e.g did:silencio